modules/android/pages/vector-search.adoc

```diff
@@ -58,12 +58,65 @@ This representation encodes how a ML model understands the input or inputs provi
 When a model considers the features of a given input as similar, the distance between the vector embeddings will be short.
 Vector embeddings are stored within embedded vector indexes.
 
+The current supported formats for Vector embeddings are:
+
+* An array of 32-bit floats.
+
+* A Base64 string that encodes the binary representation of an array of 32-bit float numbers in https://en.wikipedia.org/wiki/Endianness[Little-endian] format.
+
 === About Vector Indexes
 
 Vector Indexes are used to store and manage vector representations of content in the form of vector embeddings.
 You can use Vector Indexes to efficiently retrieve vectors, similar to a target vector.
 Before use, a Vector Index needs to be trained to compute the centroids and parameters for encoding the vectors. 
 Training is configured to commence automatically on the first Vector Search query when there are enough vectors to be trained based on the `minimum-training-size` configuration.
+If training is triggered during a query, the query may take longer to return results.
+If a query is executed against the index before it is trained, a full scan of the vectors will be performed. 
+
+NOTE: If there are insufficient vectors in the database for training, a warning message will be logged, indicating the required number of vectors.
+
+=== About Lazy Vector Indexes
+
+You can use Lazy Vector Indexes(lazy index) to use a ML model that is not available locally on the device and to create vector indexes without having vector embeddings in the documents.
```

Comment:
I don't think you need to i-cap lazy vector index....this is a standard industry term, not a branded feature.

also, spell out machine learning (ML)

Reply:
Done